test/Integration/IntegrationTestBase.cs

```diff
@@ -74,10 +74,69 @@ public class IntegrationTestBase : IDisposable
         public IntegrationTestBase(ITestOutputHelper output = null)
         {
             this.TestOutput = output;
+            this.CopyLatestSqlExtensionToBundle();
             this.SetupDatabase();
             this.StartAzurite();
         }
 
+        /// <summary>
+        /// Copy the latest build of the SqlExtension dll to the Azure Functions extension bundle.
+        /// </summary>
+        private void CopyLatestSqlExtensionToBundle()
+        {
+            string extensionBundleBinPath = Path.Combine(this.GetExtensionBundlePath(), "bin");
+            string sqlDllPath = Path.Combine(GetPathToBin(), "Microsoft.Azure.WebJobs.Extensions.Sql.dll");
+            File.Copy(sqlDllPath, Path.Combine(extensionBundleBinPath, "Microsoft.Azure.WebJobs.Extensions.Sql.dll"), true);
```

Comment:
The more I think about it the less I like doing this - silently overwriting bundle contents could easily cause issues when trying to test/verify stuff locally. Especially for people who may not know this happened - if someone comes in and runs the tests they could easily get in a bad state and not have any idea what happened. 

What about instead talking with the core tools team and seeing if they'd be open to adding an environment  variable and/or command line parameter to specify what bundle to use? This would even be helpful for testing bundles in general so seems like an overall good thing to add.

And in the meantime I feel like having the manual instructions might actually be fine. Or another couple options that would make this better : 

1. Have a script that users can run that copies it over - this way it's a deliberate action so they're aware its happening
2. Regardless, having it copy the original DLL to *.bak so it's easy to go back isn't a bad idea (you can also just delete the entire folder - that's what I do now - but it wouldn't be a bad idea regardless so it's easy to see when it's been replaced)

Reply:
I agree, I'll reach out to the core tools team.

In the meantime I'll add a section to our test readme and create another PR for adding a script.